multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/flowable/ProcessActionTest.java

```diff
@@ -0,0 +1,99 @@
+package org.cloudfoundry.multiapps.controller.process.flowable;
+
+import static org.mockito.ArgumentMatchers.anyString;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import org.cloudfoundry.multiapps.controller.core.cf.CloudControllerClientProvider;
+import org.cloudfoundry.multiapps.controller.core.persistence.service.ProgressMessageService;
+import org.flowable.engine.HistoryService;
+import org.flowable.engine.ProcessEngine;
+import org.flowable.engine.RuntimeService;
+import org.flowable.engine.runtime.Execution;
+import org.flowable.variable.api.history.HistoricVariableInstanceQuery;
+import org.junit.jupiter.api.BeforeEach;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+abstract class ProcessActionTest {
+
+    static final String EXECUTION_ID = UUID.randomUUID()
+                                           .toString();
```

Comment:
When having this many identical function calls, I suggest making a supplier of random uuids and calling its `get()` method